versioned_docs/version-24.04/installation/installation-of-centreon-ha/overview.md

```diff
@@ -1,8 +1,34 @@
 ---
 id: overview
-title: Centreon HA
+title: How Centreon HA works
 ---
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-The Centreon HA documentation is being reworked. If you need to set up HA in 24.04, please contact Centreon Professional Services, or your Centreon partner.
+## How does a Centreon HA cluster work?
+
+In the central cluster, all Centreon processes ("resources") are managed by the clustering tools (Pacemaker and Corosync).
+
+1. Everything is OK : central node 1 (the current active node) receives data from the pollers, and all relevant files are synchronized by a dedicated script  (**centreon_central_sync**) onto central node 2 (the current passive node) so that the passive node is ready to become the active node at all times.
+2. An incident occurs and central node 1 (the active node) goes down.
+   - Corosync detects that it is down: after consulting with the quorum device, it tells central node 2 to become the active node.
+   - An operator is notified that central node 1 is down, thanks to the Centreon monitoring that has been set up on the poller.
+3. Central node 2 is now the active node. It receives data from the pollers. During this time, the operator tries to understand why central node 1 is down. They must fix the problem using the cluster management tool **pcs**, not by manipulating central node 1 directly.
```

Comment:
> They must fix the problem using the cluster management tool **pcs**, not by manipulating central node 1 directly.

This is misleading: they may fix the problem the way they want (as the problem could be a full disk or a broken network card, for instance), but they must not restart centreon processes manually and use pcs for restarting them.